tests/pytest/test_unique_clear.py

```diff
@@ -4,15 +4,34 @@ def test_unique_clears(testdir):
 
     testdir.makepyfile(
         """
-        def test_generate_two_unique_booleans(faker):
+        import pytest
+        from faker.exceptions import UniquenessException
+
+        NUM_SAMPLES = 100
```

Comment:
We already have a `num_samples` fixture for this purpose. You could have just copy pasted the code snippet I wrote. Did you encounter issues?

Reply:
the `num_samples` fixture isn't part of the fixtures that the faker pytest plugin exposes, so I just defined it there to stay within the spirit of the code sample.

Reply:
While the `num_samples` fixture is not part of the pytest plugin `Faker` ships with, there is one defined for the test suite. I however misremembered, and the fixture is only available under `tests.providers` as defined [here](https://github.com/joke2k/faker/blob/565d3aa016cb986031ad00746c4d3066e9743710/tests/providers/conftest.py#L36). This should be OK for now.